packages/core/src/verify.ts

```diff
@@ -0,0 +1,166 @@
+import { IgnitionError } from "./errors";
+import { ERRORS } from "./errors-list";
+import { builtinChains } from "./internal/chain-config";
+import { FileDeploymentLoader } from "./internal/deployment-loader/file-deployment-loader";
+import { encodeDeploymentArguments } from "./internal/execution/abi";
+import { loadDeploymentState } from "./internal/execution/deployment-state-helpers";
+import { DeploymentState } from "./internal/execution/types/deployment-state";
+import { ExecutionResultType } from "./internal/execution/types/execution-result";
+import {
+  DeploymentExecutionState,
+  ExecutionSateType,
+} from "./internal/execution/types/execution-state";
+import { assertIgnitionInvariant } from "./internal/utils/assertions";
+import { findExecutionStatesByType } from "./internal/views/find-execution-states-by-type";
+import { Artifact, BuildInfo } from "./types/artifact";
+import {
+  ChainConfig,
+  SourceToLibraryToAddress,
+  VerifyResult,
+} from "./types/verify";
+
+/**
+ * Retrieve the information required to verify all contracts from a deployment on Etherscan.
+ *
+ * @param deploymentDir - the file directory of the deployment
+ * @param customChains - an array of custom chain configurations
+ * @param libraryInfo - an object containing library names and addresses
```

Comment:
```suggestion
```

Reply:
(this UI can be a bit confusing, it's a deletion because we don't have that param anymore)